docs/get-started/with-python.md

```diff
@@ -28,20 +28,20 @@ Use the CPU package if you are running on Arm CPUs and/or macOS.
 pip install onnxruntime
 ```
 
-### Install ONNX Runtime GPU (CUDA 11.x)
+### Install ONNX Runtime GPU (CUDA 12.x)
 
-The default CUDA version for ORT is 11.8.
+The default CUDA version for ORT is 12.2.
```

Comment:
I suggest to add a section for previous versions similar to https://pytorch.org/get-started/previous-versions/, and add command line for installing each version. Otherwise, it is hard to install older version.

Reply:
that's a good suggestion

Reply:
we should do that as a separate task.